teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<ul><li>Delete newline before 'Proteochemometrics (PCM)'</li><li>Double check that ideally list here is identical with header in theory and practical part, I only spotted this one: Data splitting methods vs. Splitting methods</li><li>tbd: maybe rearrange to start with a definition of what PCM is?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Good suggestion about rearrangement (here and in next comments). I will move PCM to the very top and add the rest of sections (i.e. 'Data preparation', 'Machine learning principles', and 'Applications [...]') as subsections. </p>

Reply:
Changed according to Dominique's suggestion instead.